ui/partials/perf/graphsctrl.html

```diff
@@ -23,11 +23,20 @@
           </td>
         </tr>
       </table>
-      <button class="btn btn-primary" ng-click="addTestData()">
+      <button class="btn btn-primary" ng-click="addTestData()" ng-hide="seriesList.length == 0">
         <span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Add more test data</button>
     </div>
-    <div id="data-display">
-      <select ng-model="myTimerange" ng-options="timerange.text for timerange in timeranges track by timerange.value" ng-change="timeRangeChanged()">
+        <div class="panel-center-layout">
+          <div class="panel panel-info" ng-show="seriesList.length == 0">
+            <div class="panel-heading">No graphs date</div>
+            <div class="panel-content">
+            <p >Without any graphs been specified</p>
+            <button type="button" class="btn btn-primary btn-lg btn-block" ng-click="addTestData()"> Add more test </button>
```

Comment:
I haven't tested the branch, but at least looking at the markup it seems these two lines should be indented as they are child elements of the parent `<div>`?